x-pack/plugin/sql/qa/src/main/resources/date.csv-spec

```diff
@@ -2,6 +2,73 @@
 // Date
 //
 
+currentDateKeywordWithDivision
+SELECT YEAR(CURRENT_TIMESTAMP) / 1000 AS result;
+
+    result
+---------------
+2
+;
+
+currentDateFunctionNoArgsWithDivision
+SELECT YEAR(CURRENT_TIMESTAMP()) / 1000 AS result;
+
+    result
+---------------
+2
+;
+
+todayWithDivision
+SELECT YEAR(TODAY()) / 1000 AS result;
+
+    result
+---------------
+2
+;
+
+todayIntervalSubstraction
+SELECT YEAR(TODAY() - INTERVAL 2 YEARS) / 1000 AS result;
```

Comment:
Here I would test this differently so that the result of the query is not the test above it, meaning subtracting 20+ years (to get into 1900s) and using a TRUNCATE without parameter to get a result of `1`.